modules/rte_mis_student/config/install/field.field.mini_node.student_details.field_mother_aadhar_number.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - eck.eck_type.mini_node.student_details
+    - field.storage.mini_node.field_mother_aadhar_number
+id: mini_node.student_details.field_mother_aadhar_number
+field_name: field_mother_aadhar_number
+entity_type: mini_node
+bundle: student_details
+label: "Mother's Aadhar Number"
+description: "Student's Mother's Aadhar Card Number"
```

Comment:
Same here Description should mention to add the last 4 digit number of aadhaar 
Why is this field of type string? user can enter string values?